files/plugins/connector/redis/mcollective/connector/redis.ddl

```diff
@@ -0,0 +1,9 @@
+metadata    :name        => "Redis Connector",
+            :description => "Connector plugin for Redis middleware",
+            :author      => "R.I.Pienaar <rip@devco.net>",
+            :license     => "ASL 2.0",
+            :version     => "1.0.0",
+            :url         => "https://github.com/ripienaar/mcollective-vagrant",
```

Comment:
Is this url correct? It does not seem to be related to the redis connector.

Reply:
I couldn't find another or "the" source, but the redis connector in both  mcollective modules are the same. So the voxpupuli one and the one from https://github.com/ripienaar/mcollective-vagrant/tree/master/deploy/modules/mcollective/files/lib/connector are the same.

this is only a meta file, but i don't wanted to write my name in there, i only wanted it to work ;-)